client/pages/match-details.jsx

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import React from 'react';
+import formatDate from './format-date';
+const MatchDetails = props => {
+  const teamId = {
+    leagueId: props.fixtures.league.id,
+    currentSeason: props.fixtures.league.season,
+    awayId: props.fixtures.teams.away.id,
+    homeId: props.fixtures.teams.home.id,
+    date: props.fixtures.fixture.date.slice(0, 10),
+    utcDate: props.fixtures.fixture.date
+  };
+  axios.post('/api/team-form/:teamId', { teamId }).then(response => {
+  }).catch(err => {
+
+    console.error(err);
+  });
```

Comment:
You shouldn't request data like this in a function component's body. Convert this to a class component and fetch the data in `componentDidMount` instead.